platform-connectors/Tiltfile

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-docker_build(
-    "ghcr.io/nvidia/nvsentinel-platform-connectors",
-    context="..",
-    dockerfile="./Dockerfile"
+# Build and deploy
+custom_build(
```

Comment:
just for my understanding does `ko_build` https://github.com/tilt-dev/tilt-extensions/tree/master/ko not work for us? Is it due to our complex module structure?